mappings/net/minecraft/client/render/BufferBuilder.mapping

```diff
@@ -81,25 +84,25 @@ CLASS net/minecraft/class_287 net/minecraft/client/render/BufferBuilder
 	METHOD method_43578 build ()Lnet/minecraft/class_287$class_7433;
 	METHOD method_43579 resetBuilding ()V
 	METHOD method_43580 releaseBuiltBuffer ()V
-	CLASS class_4574 DrawArrayParameters
+	CLASS class_4574 DrawParameters
 		METHOD method_31957 getIndexBufferStart ()I
 		METHOD method_31958 getIndexBufferEnd ()I
 		METHOD method_31961 getIndexBufferLength ()I
 		METHOD method_43429 getVertexBufferPosition ()I
 		METHOD method_43430 getVertexBufferLimit ()I
 		METHOD method_43431 getIndexBufferPosition ()I
 		METHOD method_43432 getIndexBufferLimit ()I
-	CLASS class_5594 State
+	CLASS class_5594 TranslucencySortingData
```

Comment:
transparent is to be the much established term for this kind or rendering, applies to the ChunkBuilder change below too